members/C001095.yaml

```diff
@@ -2,61 +2,55 @@ bioguide: C001095
 contact_form:
   method: post
   steps:
-    - visit: "http://www.cotton.senate.gov/?p=contact"
+    - visit: "https://www.cotton.senate.gov/contact/contact-tom"
     - fill_in:
         - name: First Name
-          selector: "#firstName"
+          selector: "#input-B03E05C4-4040-F985-52CD-9CA44623DE5E"
```

Comment:
can we use selectors that are less generated looking?

Reply:
I wasn't able to find/use other selectors that were more general instead of these ids